haas/ext/auth/database.py

```diff
@@ -131,6 +131,21 @@ def user_remove_project(user, project):
     db.session.commit()
 
 
+@rest_call('PUT', '/auth/basic/user/<user>/set_privilege', Schema({
+    'user': basestring,
+    'privilege': bool,
```

Comment:
This is inconsistent with the docs, which say this should be either the string `admin` or `regular`. As outlined above I actually think it should stay a boolean and you should update the docs.